include/kitepp/kite/internal.hpp

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+ *  SPDX-License-Identifier: MIT
+ *
+ *  Copyright (c) 2020-2022 Bhumit Attarde
+ *
+ *  Permission is hereby  granted, free of charge, to any  person obtaining a
+ * copy of this software and associated  documentation files (the "Software"),
+ * to deal in the Software  without restriction, including without  limitation
+ * the rights to  use, copy,  modify, merge,  publish, distribute,  sublicense,
+ * and/or  sell copies  of  the Software,  and  to  permit persons  to  whom the
+ * Software  is furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE  IS PROVIDED "AS  IS", WITHOUT WARRANTY  OF ANY KIND,  EXPRESS
+ * OR IMPLIED,  INCLUDING BUT  NOT  LIMITED TO  THE  WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR  A PARTICULAR PURPOSE AND  NONINFRINGEMENT. IN
+ * NO EVENT  SHALL THE AUTHORS  OR COPYRIGHT  HOLDERS  BE  LIABLE FOR  ANY
+ * CLAIM,  DAMAGES OR  OTHER LIABILITY, WHETHER IN AN ACTION OF  CONTRACT, TORT
+ * OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  OR THE
+ * USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#pragma once
+
+#include <functional>
+#include <type_traits>
+
+#include "../kite.hpp"
+#include "../utils.hpp"
+
+namespace kiteconnect {
+
+inline string kite::getAuth() const { return authorization; }
+
+inline string kite::encodeSymbolsList(const std::vector<string>& symbols) {
+    string str;
+    for (const auto& symbol : symbols) { str.append(FMT("i={0}&", symbol)); };
```

Comment:
This `symbol` has to be URL encoded first. Otherwise it will fail for weird symbols like `M&M`.

Reply:
check now?

Reply:
Looks alright